### PR TITLE
Committing the new test case

### PR DESCRIPTION
**Pull request by upgradgit2**

### CODE REVIEW

File: src/test/java/com/practice/ShoppingCartTest.java [hunk 1]
```diff
@@ -68,4 +68,16 @@ void shouldCalculateTotalPrice() {
...
 
     }
+
+    @DisplayName("Calculation Of Total Item Price In Cart1")
+    void shouldCalculateTotalPrice2() {
+        // given
+        cart.addItem(item1);
+        cart.addItem(item2);
+
+        double totalPrice = cart.calculateTotalPrice();
```
Comment (algdsapractice) [inline src/test/java/com/practice/ShoppingCartTest.java:78]: Please add necessary comments

File: src/test/java/com/practice/ShoppingCartTest.java [hunk 2]
```diff
@@ -68,4 +68,16 @@ void shouldCalculateTotalPrice() {
...
+        double expectedPrice = 120000.0;
+        assertEquals(expectedPrice, totalPrice, "Price Not Matching");
+
```
Comment (algdsapractice) [inline src/test/java/com/practice/ShoppingCartTest.java:82]: Elaborate on message

**Review by algdsapractice — CHANGES_REQUESTED** [review]

File: src/test/java/com/practice/ShoppingCartTest.java (hunk 1 above)
Comment (upgradgit2) [reply src/test/java/com/practice/ShoppingCartTest.java:78]: addressed

File: src/test/java/com/practice/ShoppingCartTest.java (hunk 2 above)
Comment (upgradgit2) [reply src/test/java/com/practice/ShoppingCartTest.java:82]: addressed

**Pull request merged**